youtube_dl/extractor/bilibili.py

```diff
@@ -113,7 +113,7 @@ def _real_extract(self, url):
                 self._report_error(js)
             cid = js['result']['cid']
 
-        payload = 'appkey=%s&cid=%s&otype=json&quality=2&type=mp4' % (self._APP_KEY, cid)
+        payload = 'appkey=%s&cid=%s&otype=json&quality=0&type=flv' % (self._APP_KEY, cid)
```

Comment:
1. Both should be extracted.
2. Update tests.

Reply:
> 1. Both should be extracted.

https://github.com/rg3/youtube-dl/issues/12095#issuecomment-280003212

Reply:
2. Tests have been updated.